### ISSUE TITLE
ASIHPMixedTank test file fails on linux

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#5845]
## Issue overview

ASIHPMixedTank is failing in the linux integration tests, runs to completion on other platforms.  @rraustad identified some questionable input [here](https://github.com/NREL/EnergyPlus/pull/5831#issuecomment-242728106) but that doesn't explain the test failure.

CI output:

```
EnergyPlus Starting
EnergyPlus, Version 8.6.0-a17aa64355, YMD=2016.08.24 17:50
Processing Data Dictionary
Processing Input File
Initializing Simulation
Test Failed
```
### Details

Some additional details for this issue (if relevant):
- Platform (linux CI)
- Version of EnergyPlus (v8.6 develop a17aa643559142a5ed22a08ecc7aa8a015d8bbba)
### Checklist

Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
- [ ] Defect file added (list location of defect file here)
- [ ] Ticket added to Pivotal for defect (development team task)
- [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by mjwitte** [comment]
@Myoldmopar @mbadams5 @shenb Can one of you reproduce this failure on linux so we can get more details on why it's failing?

**Comment by shenb** [comment]
I don’t have a linux system on my computer.

**Comment by Myoldmopar** [comment]
It's because of [this line](https://github.com/NREL/EnergyPlus/blob/11c622443d7563d1dcd1c867263093809b94fa2a/src/EnergyPlus/WaterThermalTanks.cc#L5639-L5641).  When it gets to that point in the code for `VSCoilID==2`, the array of `MSRatedWaterVolFlowRate` values are all zero, leading to a divide by zero.  Any thoughts @rraustad @shenb or anyone else?  I'll try to keep looking...

**Comment by rraustad** [comment]
Check to see what happens when [this](https://github.com/NREL/EnergyPlus/blob/11c622443d7563d1dcd1c867263093809b94fa2a/src/EnergyPlus/IntegratedHeatPump.cc#L668) line is hit.

**Comment by rraustad** [comment]
May have linked the wrong line in code, I think it's [this](https://github.com/NREL/EnergyPlus/blob/11c622443d7563d1dcd1c867263093809b94fa2a/src/EnergyPlus/IntegratedHeatPump.cc#L689) one.

**Comment by shenb** [comment]
Edwin and Rich,

Thanks for help identifying the bug in Linux. I will make the fix today.

Bo

**Comment by shenb** [comment]
Edwin,

I made the bug fix and started a pull request as below,

https://github.com/NREL/EnergyPlus/pull/5858

thanks,

Bo

**Comment by mjwitte** [comment]
Closed via #5858
